packages/lisk-p2p/src/validation.ts

```diff
@@ -67,7 +67,7 @@ export const validatePeerInfo = (
 	const version = protocolPeer.version;
 	const wsPort = +protocolPeer.wsPort;
 	const os =
-		protocolPeer.os && isAlpha(protocolPeer.os.toString())
+		protocolPeer.os
```

Comment:
should't we check `isAlpha` here?

Reply:
Right now in core v1, the os string contains dots and dashes and version numbers.